### PR TITLE
Improve README and install instructions

### PR DESCRIPTION
**Pull request by runnersaw**
1. Improved README
2. Improved install (move to using only environment.yml file, with OS detection to use the correct yml file)
3. Remove cv2 from all code. Use PIL.Image or ffmpeg if needed.

### CODE REVIEW

**Review by jabb1123 — APPROVED** [review]
Looks awesome. Good to remove OpenCV from GUI too!

File: application/app.py [hunk 1]
```diff
@@ -377,26 +380,45 @@ def homography_compute(self):
...
+            self.unitPixRatio,\
+            self.ui.homography_aerialview.list_points(),\
+            self.ui.homography_cameraview.list_points())
+
+        if not success:
+            self.show_error(err)
+            return
+
+        # Finally, upload to server
+        success, err = api.configHomography(\
```
Comment (dcelik) [inline application/app.py]: wat.

File: application/app.py [hunk 2]
```diff
@@ -437,18 +449,29 @@ def homography_display_results(self):
...
-            cv2.circle(videoImg, tuple(np.int32(np.round(self.videoPts[i]))), 2, cvBlue)
-            cv2.circle(videoImg, tuple(np.int32(np.round(projectedWorldPts[i]))), 2, cvRed)
-            cv2.putText(videoImg, str(i+1), tuple(np.int32(np.round(self.videoPts[i]) + 5)), cv2.FONT_HERSHEY_PLAIN, 2., cvBlue, 2)
+            videoDraw = ImageDraw.Draw(videoImg)
+            x, y = tuple(np.int32(np.round(self.videoPts[i])))
+            videoDraw.ellipse((x-r, y-r, x+r, y+r), outline=cvBlue)
+            x, y = tuple(np.int32(np.round(projectedWorldPts[i])))
+            videoDraw.ellipse((x-r, y-r, x+r, y+r), outline=cvRed)
+            x, y = tuple(np.int32(np.round(self.videoPts[i]) + 5))
+            videoDraw.text((x, y), str(i+1), fill=cvBlue)
```
Comment (dcelik) [inline application/app.py]: I think differentiating the text color would be useful. White text with a black stroke is visible on anything.

**Review by dcelik — CHANGES_REQUESTED** [review]
Looks go to me!

**Review by dcelik — APPROVED** [review]
Very interesting way of creating circles and texts! LGTM

**Pull request merged**